assets/js/modules/reader-revenue-manager/components/dashboard/PublicationApprovedOverlayNotification.js

```diff
@@ -116,6 +123,15 @@ function PublicationApprovedOverlayNotification() {
 		dismissOverlayNotification(
 			RRM_PUBLICATION_APPROVED_OVERLAY_NOTIFICATION
 		);
+
+		// If this notification is displayed on succcessful module setup, dismiss accordingly.
+		if (
+			notification === 'authentication_success' &&
+			slug === READER_REVENUE_MANAGER_MODULE_SLUG
+		) {
+			setNotification( undefined );
+			setSlug( undefined );
+		}
```

Comment:
Unless I've missed a reason this needs to be here, I think it would be easier to follow if we dismissed the setup notification at the point where it triggers showing this overlay.

```suggestion
```

Reply:
Good point, updated, thank you!